.github/workflows/downstream_tests.yml

```diff
@@ -435,7 +435,7 @@ jobs:
         run: |
           cd pointblank
           uv pip install --system ".[dev]"
-          uv pip install pytest pytest-cov pytest-snapshot pandas polars "ibis-framework[duckdb,mysql,postgres,sqlite]>=9.5.0" chatlas --system    
```

Comment:
We also need `shiny` : https://github.com/posit-dev/pointblank/blob/main/.github/workflows/ci-tests.yaml#L32C131-L32C136